tests/test_models.py

```diff
@@ -0,0 +1,886 @@
+import os
```

Comment:
Missing module docstring.

Reply:
None of the tests currently contain a module docstring - is this something we want to add? How about the GPL license on tests?

Reply:
Added module docstrings to tests in [this commit](https://github.com/TIA-Lab/tiatoolbox/pull/98/commits/3758a9f6916dcbf8ca228dbb8511cf0c63502152)